event-sourcing/event-source/src/main/java/uk/gov/justice/services/eventsourcing/source/core/exception/AggregateCreationException.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.services.eventsourcing.source.core.exception;
+
+public class AggregateCreationException extends RuntimeException {
+
+    public static final String ACCESS_ERROR = "Unable to create aggregate due to access error";
```

Comment:
Doing this way i believe it was easier for me to differentiate between two different type of exception and also validate the test exception messages
 expectedException.expectMessage(AggregateCreationException.ACCESS_ERROR);